perun-core/src/test/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_user_facility_attribute_def_virt_defaultUnixGIDTest.java

```diff
@@ -57,7 +58,8 @@ public void setUp() throws Exception{
 				session = mock(PerunSessionImpl.class, RETURNS_DEEP_STUBS);
 				attrDef = session.getPerunBl().getAttributesManagerBl().getAttributeDefinition(session, AttributesManager.NS_USER_FACILITY_ATTR_VIRT + "defaultUnixGID");
 				dDefaultUnixGid = new Attribute();
-				namespace = "test";
+				gidNamespace = "test";
+				groupNameNamespace = "test";
```

Comment:
Try to use different namespace names. Using the same name could hide some bugs.

Reply:
Changed.